hack/make-release-artifacts.sh

```diff
@@ -46,7 +46,7 @@ done < <(find "${bin_dir}" -print0)
 echo >&2 "Creating ${krew_tar_archive} archive."
 (
   cd "${bin_dir}"
-  GZIP=-n tar czvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
+  tar -I "gzip -${n}" -cvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
```

Comment:
Also, we need a tester with an OSX system (which uses BSD tar). The dash options are just GNU tar and it could be possible that your change breaks the BSD tar.